app/models/site.rb

```diff
@@ -7,10 +7,48 @@ class Site < ActiveRecord::Base
   accepts_nested_attributes_for :account, update_only: true
 
   class << self
-    delegate :account, :application_name, :institution_name, :institution_name_full, :reload, :update, to: :instance
+    delegate :account, :application_name, :institution_name,
+             :institution_name_full, :reload, :update, :announcement_text,
+             :marketing_text, :featured_researcher, to: :instance
 
     def instance
       first || create
     end
   end
+
+  def announcement_text
```

Comment:
IMHO these methods do not belong on the site model.  You should extract this stuff into a Form object which is responsible for saving the site and any content blocks.